tests/10-vcdm2.js

```diff
@@ -608,26 +604,20 @@ describe('Validity Period', function() {
         this.test.link = `https://w3c.github.io/vc-data-model/#validity-period:~:text=If%20a%20validFrom%20value%20also%20exists%2C%20the%20validUntil%20value%20MUST%20express%20a%20datetime%20that%20is%20temporally%20the%20same%20or%20later%20than%20the%20datetime%20expressed%20by%20the%20validFrom%20value.`;
         const positiveTest = require(
           './input/credential-validUntil-validFrom-ok.json');
-        positiveTest.validFrom = createTimeStamp({skew: -2});
-        positiveTest.validUntil = createTimeStamp({skew: 2});
+        positiveTest.validFrom = createTimeStamp({skew: -1 * oneYear});
+        positiveTest.validUntil = createTimeStamp({skew: oneYear});
         await assert.doesNotReject(endpoints.issue(positiveTest),
           'Failed to accept a VC with a `validUntil` after its `validFrom`.');
         const negativeTest = require(
           './input/credential-validUntil-validFrom-fail.json');
-        negativeTest.validFrom = createTimeStamp({skew: 2});
-        negativeTest.validUntil = createTimeStamp({skew: -2});
-        let error;
-        let result;
-        try {
-          result = await endpoints.issue(negativeTest);
-        } catch(e) {
-          error = e;
-        }
-        if(error) {
-          return;
-        }
-        await assert.rejects(endpoints.verify(result),
-          'Failed to reject a VC with a `validUntil` before its `validFrom`.');
+        negativeTest.validFrom = createTimeStamp({skew: oneYear});
```

Comment:
Change the API as you suggested in https://github.com/w3c/vc-data-model-2.0-test-suite/pull/96/files#r1691824279 and I think this PR is good to go.

Such as...
```suggestion
        negativeTest.validFrom = createTimeStamp({skewYears: 1});
```

Reply:
Sorry for not taking the suggested change made all the changes here: https://github.com/w3c/vc-data-model-2.0-test-suite/pull/96/commits/5ddd574c83cf707a158bd7826238606cdef1fd63 and excellent feedback on this PR.